README.md

```diff
@@ -27,39 +33,188 @@
   </a>
 </p>
 
-## Backstory
-
-...
-
 ## Key features
 
 - 🍱 Reliable serialization and deserialization
 - 🔐 Type safety with autocompletion
 - 🐾 Negligible runtime footprint
 - 💫 Framework agnostic
 
-## Example
+
+## Backstory
+
+At [Blitz](https://github.com/blitz-js/blitz), we have struggled with the limitations of JSON. Modern databases often store complex types, like Date objects, but it impossible to easily `stringify` your responses without converting any object types to strings. Moreover, you often have to keep track of which fields you need to convert back to objects in your client!
+
+Superjson solves these issues by providing a thin wrapper over `JSON.stringify` and `JSON.parse`. Look at the difference between manually converting invalid fields, and using Superjson to handle this for you:
+
+
+```js
+// 😔 without superjson
+
+// retrieve user object
+const user = await db.user.findOne();
+
+/* 
+user = {
+  id: 1,
+  createdAt: new Date(1),
+  updatedAt: new Date(2),
+  name: undefined,
+  email: 'me@here.com',
+  posts: [
+    {
+      id: 1,
+      createdAt: new Date(3),
+      updatedAt: new Date(4),
+      body: 'hello world!',
+    },
+  ],
+}; 
+*/
+
+// manually update invalid fields 
+const serializableUser = {
+  ...user,
+  createdAt: user.createdAt.toIsoString(),
+  updatedAt: user.updatedAt.toIsoString(),
+  name: 'undefined',
+  posts: user.posts.map(post => ({
+    ...post,
+    createdAt: post.createdAt.toIsoString(),
+    updatedAt: post.updatedAt.toIsoString(),
+  })),
+};
+
+// api response
+return JSON.stringify(serializableUser);
+
+// fetch user from api response
+const res = await fetch('/api/user');
+const json = await res.json();
+
+// restore fields
+const user = {
+  ...res,
+  createdAt: new Date(res.createdAt),
+  updatedAt: new Date(res.updatedAt),
+  name: undefined,
+  posts: res.posts.map(post => ({
+    ...post,
+    createdAt: new Date(post.createdAt),
+    updatedAt: new Date(post.createdAt),
+  })),
+};
+```
+
+```js
+// ✨ with superjson
+
+// retrieve user object
+const user = await db.user.findOne();
+
+/* 
+user = {
+  id: 1,
+  createdAt: new Date(1),
+  updatedAt: new Date(2),
+  name: undefined,
+  email: 'me@here.com',
+  posts: [
+    {
+      id: 1,
+      createdAt: new Date(3),
+      updatedAt: new Date(4),
+      body: 'hello world!',
+    },
+  ],
+}; 
+*/
+
+// api response
+return superjson.stringify(serializableUser);
+
+// fetch user from api response
+const res = await fetch('/api/user');
+
+// restore fields
+const user = superjson.parse(res);
+```
+
+## Getting started
+
+Install the library with your package manager of choice, e.g.:
+
+```
+yarn add superjson
+```
+
+## Usage
+
+The simplest way of using `superjson` is with its `stringify` and `parse` functions. If you know how to use `JSON.stringify` in JavaScript, you already know Superjson!
+
+Simply stringify any expression you’d like:
+
+```js
+import { stringify } from 'superjson';
```

Comment:
```suggestion
import SuperJSON from 'superjson';
```

Maybe do this and then `SuperJSON.stringify` to make the similarities to `JSON.stringify` obvious

Reply:
Tbh, I don't think we need to include the `import` line in the example. That one should be self-explanatory :D